src/Compute/Compute/VirtualMachine/Config/NewAzureVMConfigCommand.cs

```diff
@@ -247,6 +257,29 @@ public override void ExecuteCmdlet()
                 }
             }
 
+            if (this.IsParameterBound(c => c.EncryptionIdentity))
```

Comment:
Will this return error if encryption identity is not part of identity getting assigned to the VM?

Reply:
No it won't return any error if encryption identity is not a part of identities assigned to the VM ?

Should I add error check here ?

Reply:
okk. Lets leave it as it is now and get feedback from powershell team during review.

Reply:
Understood